pkgs/development/python-modules/ntlmlib/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  # deps
+  cryptography,
+  ...
+}:
+
+buildPythonPackage rec {
+  pname = "ntlmlib";
+  version = "0.72";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-azZQJ+w82vCUIaZ5pbKzl6i0H4rd6NzYi4DmlajS6cI=";
+  };
+
+  propagatedBuildInputs = [
+    cryptography
+  ];
+
+  patchPhase = ''
+    # remove references to 'ordereddict' (deprecated)
+    sed -i "30d" setup.py
+    sed -i "21,24c from collections import OrderedDict" ntlmlib/messages.py
+  '';
+
+  meta = {
+    description = "A robust, fast and efficient ‘first-class’ Python Library for NTLM authentication, signing and encryption.";
+    license = lib.licenses.asl20;
+    homepage = "https://github.com/ianclegg/ntlmlib/";
+    maintainers = [ ];
```

Comment:
Consider adding yourself to maintainers.

Reply:
Good to go now